ioctl/newcmd/alias/aliasset_test.go

```diff
@@ -18,6 +18,7 @@ import (
 )
 
 func TestNewAliasSetCmd(t *testing.T) {
+	require := require.New(t)
 	ctrl := gomock.NewController(t)
 	cfg := config.Config{
 		Aliases: map[string]string{
```

Comment:
add Wallet = tempDir . because this command use client.SetAlias to write default config file.